datasetinsights/datasets/groceries_real.py

```diff
@@ -358,17 +358,16 @@ def download(self):
         """Download dataset from GCS
         """
         cloud_path = f"gs://{const.GCS_BUCKET}/{self.GCS_PATH}"
+        client = GCSClient()
         # download label file
-        label_zip = download_file_from_gcs(
-            cloud_path, self.root, self.LABEL_ZIP
-        )
+        label_zip = self.LABEL_ZIP
+        client.download(url=cloud_path, local_path=self.root)
         with zipfile.ZipFile(label_zip, "r") as zip_dir:
```

Comment:
You can archive the downloaded file and use `shutil.unpack_archive`, check details in this [PR](https://github.com/Unity-Technologies/datasetinsights/pull/57/files#).

Reply:
Dataset won't download anything going forward.